docs/source/user-guide/tasks/create-project-archive.rst

```diff
@@ -67,3 +67,43 @@ EXAMPLE::
         557  06-10-2016 10:33   iris/iris_plot/main.py
   ---------                     -------
        6003                     5 files
+
+Extracting the archive file
+===========================
+
+It is recommended that you unarchive Anaconda Project bundles using
+the ``unarchive`` command::
+
+  anaconda-project unarchive <bundle>
+
+This will unarchive any compression format (``.zip``, ``.tar.gz``, and
+``.tar.bz2``) on all supported platforms.
+
+
+Experimental: Packaging environments
+====================================
+
+There are cases where it may be preferable to package the
+Conda environments directly into the archive. For example,
+when the target system cannot connect to the repository to
+download and install packages.
```

Comment:
```suggestion
There are cases where it may be preferable to package the
Conda environments directly into the archive. For example,
you may want to support uses where the target system cannot
connect to the repository to download and install packages.
```

(wasn't a sentence before)